farizdotid/DAFTAR-API-LOKAL-INDONESIA
# feat: add apimock under utilities

## What it does

- add APIMock under utilities
- run `bun check`
- run `bun start` to regenerate README files

## How to test

1. run `bun check` to make sure there is no error
2. run `bun start` to regenerate README files
3. observe the README.md and README.en.md to check if APIMock is added.

## Screenshot

### README.en.md
![image](https://github.com/user-attachments/assets/f5c1d377-5798-4386-88a6-73934eb1765f)

### README.md
![image](https://github.com/user-attachments/assets/145c473f-c2f2-4276-8662-bfcf2415b007)
